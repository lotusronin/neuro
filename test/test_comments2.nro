import stdio;

fn main() : u32 {
    stdio::puts("before line comment");
    //stdio::puts("in line comment");
    stdio::puts("after line comment");
    //stdio::puts("in line comment2");
    return 0;
}
