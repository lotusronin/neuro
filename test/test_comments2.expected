before line comment
after line comment
